tests/suite/src/__tests__/correctness/fast/model-sync.test.ts

```diff
@@ -33,108 +33,107 @@ const testModel: ModelDefinition = {
   },
 }
 
-const FLIGHT_OPTIONS_0: ClientOptions = {
-  headers: new Array(),
-  username: undefined,
-  password: undefined,
-  token: undefined,
-  tls: false,
-  host: CeramicFlightEndpoints[0].host,
-  port: CeramicFlightEndpoints[0].port,
-}
-const FLIGHT_OPTIONS_1: ClientOptions = {
-  headers: new Array(),
-  username: undefined,
-  password: undefined,
-  token: undefined,
-  tls: false,
-  host: CeramicFlightEndpoints[1].host,
-  port: CeramicFlightEndpoints[1].port,
+interface Clients {
+  url: string,
+  ceramic: CeramicClient,
+  flight: FlightSqlClient,
+  model: ModelClient,
+  modelInstance: ModelInstanceClient,
 }
 
 
-describe('model integration test for list model and MID', () => {
```

Comment:
this test had overlap with the update test, but the update test was dynamic for the number of ceramic nodes. Now this is as well.